TruePath/AbsolutePath.cs

```diff
@@ -46,7 +46,7 @@ namespace TruePath;
     /// <param name="value">Path string to normalize.</param>
     /// <param name="checkAbsoluteness">Flag indicating whether absoluteness of path should be checked</param>
     /// <exception cref="ArgumentException">Thrown if the passed string does not represent an absolute path.</exception>>
-    private AbsolutePath(string value, bool checkAbsoluteness)
+    internal AbsolutePath(string value, bool checkAbsoluteness)
```

Comment:
If we get rid of `AbsoluteDirectory.GetCurrentDirectory()`, this change will be unnecessary.